.github/workflows/cron-testing.yml

```diff
@@ -0,0 +1,56 @@
+# Schedule testing task
+# https://docs.github.com/cn/actions/using-workflows/events-that-trigger-workflows#schedule
+# Please set SENDER_MNEMONIC, SLACK_WEBHOOK_URL in github secrets.
+
+name: Cron testing
+
+on:
+  schedule:
+    # Please set the start time (UTC time zone).
+    # https://docs.github.com/cn/actions/using-workflows/events-that-trigger-workflows#schedule
+    - cron:  '0 0 * * *'
```

Comment:
`0 0 12 ? * MON-FRI` -> which is 5AM PST every weekday.

Reply:
Fixed